stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -138,6 +138,8 @@ public void prepare() throws Exception {
               + "be at least the number of threads, preferably a multiple of it."
       ));
     }
+    // All test operations happen in mLocalpath/fuseIOStressBench
+    mParameters.mLocalPath = Paths.get(mParameters.mLocalPath, "fuseIOStressBench").toString();
```

Comment:
do we want to name the subfolder following the operation type to avoid conflicts?
but it's update to you
also, did we add cluster-id to the test path?

Reply:
Operation type you mean "Write", "LocalRead, "RemoteRead" and "ClusterRead"? 

No we don't add cluster id to the test path. We add the job-worker id though

Reply:
we can keep the path for now and no need to worry about it